lib/lutaml/xmi/parsers/xml.rb

```diff
@@ -12,39 +13,44 @@ class XML
           "0" => "C",
           "1" => "M",
         }.freeze
-        attr_reader :main_model, :xmi_cache
-
-        # @param [String] io - file object with path to .xmi file
-        #        [Hash] options - options for parsing
-        #
-        # @return [Lutaml::XMI::Model::Document]
-        def self.parse(io, _options = {})
-          new.parse(Nokogiri::XML(io.read))
+        attr_reader :main_model, :xmi_cache, :xmi_root_model
+
+        # @param xml [String] path to xml
+        # @param options [Hash] options for parsing
+        # @return [Lutaml::Uml::Document]
+        def self.parse(xml, _options = {})
+          xml_content = File.read(xml).gsub("\t", "  ")
```

Comment:
We don’t need to remove Tabs because we are not doing a compare.

Reply:
Fixed in 3fc1ecea4aa1ae2d1bf4752baca8cabe0f73d71b.